tck/src/main/java/org/eclipse/microprofile/rest/client/tck/ClientHeaderParamTest.java

```diff
@@ -28,8 +28,9 @@
 import static org.testng.Assert.fail;
 
 import java.util.Arrays;
-import java.util.Map;
 import java.util.stream.Collectors;
```

Comment:
We still need to import `Map` since it is used in `testHeaderNotSentWhenExceptionThrownAndRequiredIsFalse`

Reply:
good catch and sorry, I forgot to `git stash` the rest of the changes when checking this commit :(